stories/uv.stories.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx h */
+import { h } from 'jsx-dom';
+import { withKnobs, text, number } from '@storybook/addon-knobs';
+import { Viewer } from '../src/index';
+
+export default { title: 'Universal Viewer',   decorators: [withKnobs] };
+
+type Data = {
+  manifestUri: string,
+  configUri?: string,
+  collectionIndex?: number,
+  manifestIndex?: number,
+  sequenceIndex?: number,
+  canvasIndex?: number,
+  rangeId?: number,
+  rotation?: number,
+  xywh?: string,
+  embedded?: boolean,
+  locales?: any, //??
```

Comment:
Does this ?? comment indicate that further work/discussion is needed here?

Reply:
I think so, as far as I can tell, there are no types for the locale config in the UV.

Reply:
@edsilv, any thoughts on this? Hopefully we can either adjust the type (if necessary/possible) or remove the comment (if no further action is needed).

Reply:
I think typing it shouldn't be a problem